analyze_logs/src/analyzer.py

```diff
@@ -21,8 +22,14 @@
 class Analyzer:
   def __init__(self):
     self.log = Log()
+    self.log.info("Storage account: {0}".format(config.AZURE_STORAGE_ACCOUNT_NAME))
     self.msgQueue = Queue(account_name = config.AZURE_STORAGE_ACCOUNT_NAME, account_key=config.AZURE_STORAGE_ACCOUNT_KEY, queue_name=config.AZURE_STORAGE_QUEUE_NAME)
     self.summary = SummaryTable(config.AZURE_STORAGE_ACCOUNT_NAME, config.AZURE_STORAGE_ACCOUNT_KEY, config.AZURE_STORAGE_SUMMARY_TABLE_NAME)
+    self.keep_running = bool(config.ANALYZER_KEEP_RUNNING)
+    self.log.info("Keep running: {0}".format(self.keep_running))
+    if self.keep_running:
+      self.sleep_time = float(config.ANALYZER_SLEEP_TIME)
+      self.log.info("Sleep time between analyses: {0}".format(self.sleep_time))
```

Comment:
This should be a debug rather than an info. No biggie, just mentioning it for feedback. It can be changed after merge.